client/src/utils/api/AuthApi.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import { makeSearchParamsFromData } from '../helpers';
+
+const API_URL = process.env.REACT_APP_API_URL;
+const FORM_URLENCODED_CONTENT_TYPE = 'application/x-www-form-urlencoded;charset=UTF-8';
+
+export default class AuthApi {
+  static login(username, password) {
+    return axios({
+      url: `${API_URL}/auth/login`,
+      method: 'POST',
+      headers: {
+        'Content-Type': FORM_URLENCODED_CONTENT_TYPE,
```

Comment:
Is there any reason we are not sending JSON data and sending the username and password in the body of the request object?

Reply:
It's to comply w/ the API spec that asks for a urlencoded body for the login/register routes.

![image](https://user-images.githubusercontent.com/64167209/91058269-42950780-e5f6-11ea-958e-e1ce2df237c7.png)

I don't know the specific reaons why @f3rland designed it as such, but from experience it's not uncommon to find auth routes to have this content type, especially when they want to keep the possibility to be directly acces via the action field of a form tag or when they were in the past and then migrated to being accessed via ajax calls in a SPA.

Reply:
Dit it that way because it's a common way for auth. Can change it to make API uniform

Reply:
They can actually be passed as json object
```
POST {{host}}/auth/register HTTP/1.1
Content-Type: application/json

{
  "username": "foo@example.com",
  "password": "password"
}
```

Reply:
Reverted to plain ol' JSON and it works.
https://giphy.com/gifs/jon-stewart-the-daily-show-5aLrlDiJPMPFS